test/unit/sandbox-libraries/crypto.test.js

```diff
@@ -0,0 +1,39 @@
+describe('sandbox library - CryptoJS', function () {
+    this.timeout(1000 * 60);
+    var Sandbox = require('../../../'),
+        context;
+
+    beforeEach(function (done) {
+        Sandbox.createContext({}, function (err, ctx) {
+            context = ctx;
+            done(err);
+        });
+    });
+
+    afterEach(function () {
+        context.dispose();
+        context = null;
+    });
+
+    it('must exist', function (done) {
+        context.execute(`
+            var assert = require('assert');
+
+            assert.strictEqual(typeof CryptoJS, 'object', 'typeof CryptoJS must be object');
+            assert.strictEqual(typeof CryptoJS.AES.encrypt, 'function', 'typeof CryptoJS.AES.encrypt must be function');
+            assert.strictEqual(typeof CryptoJS.AES.decrypt, 'function', 'typeof CryptoJS.AES.decrypt must be function');
+        `, done);
+    });
+
+    it('must have basic functionality working', function (done) {
```

Comment:
Crypto usually have issues with getting random numbers and random bytes due to restrictions of random number generators. Can you add tests for them?